SConstruct

```diff
@@ -1021,43 +1021,47 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
 
 # Python 2 Package Settings
 cython_min_version = LooseVersion('0.23')
+numpy_min_version = LooseVersion('1.6.1')
 env['install_python2_action'] = ''
 if env['python_package'] == 'new':
+    print ("WARNING: The 'new' option for the Python package is "
```

Comment:
Can we deprecate it for removal at an indefinite point in the future then? I want to discourage people from using it, although the case you mention is useful. Something like `The 'new' option is deprecated and will be removed in the future` At some point, I think it would be good to strip out all the mentions of the old interface to avoid confusion, and we're 2 versions past the switch now...

Stripping out all the old mentions might be a fools errand though, since people still post on the UG about 1.7 (ugh)...